.github/workflows/tests.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
     steps:
     - name: Cancel previous
-      uses: styfle/cancel-workflow-action@89f242ee29e10c53a841bfe71cc0ce7b2f065abc #0.9.0
+      uses: styfle/cancel-workflow-action@a40b8845c0683271d9f53dfcb887a7e181d3918b #0.9.0
```

Comment:
@willnorris - that seems odd that dependabot didn't update the `#0.9.0` to `#0.9.1`, doesn't it?

Reply:
not really.  The comment is a style I typically use, but it not officially supported by dependabot.  Maybe I'll file that as a feature request.  I'm just happy they at least recognize that we prefer full SHAs, and include that in the update.  I *think* dependabot might accept suggested changes, so I'll try that.  But that would be a little annoying to have to do every time.

Reply:
```suggestion
      uses: styfle/cancel-workflow-action@a40b8845c0683271d9f53dfcb887a7e181d3918b #0.9.1
```

Reply:
looks like we'd have to merge suggestion ourselves.